app/views/kuroko2/workers/index.html.slim

```diff
@@ -39,3 +42,18 @@
                           role: 'button', class: 'btn btn-sm btn-default')
               - else
                 '&nbsp;
+            td
+              - if !worker.suspended
+                = button_to('Suspend',
```

Comment:
(nit) how about give confirm dialog to user?  🤔

Reply:
If user suspend worker accidentally, the user can unsuspend the worker. So I think confirm dialog is unneeded.

Reply:
In my case, I don't want to suspend a worker accidentally, even though I know I can restore it with only one click. anyway, that's not a big problem(why I tagged `nit`) ;)

Reply:
I understand. I added confirm dialog in https://github.com/cookpad/kuroko2/pull/94/commits/eb5520ea93fde9f326e2843413c2b0a109bcbe64.

Reply:
Many thanks!! XD